semver.md

```diff
@@ -6,7 +6,7 @@ Summary
 
 Given a version number MAJOR.MINOR.PATCH, increment the:
 
-1. MAJOR version when you make incompatible API changes,
+1. MAJOR version when you make incompatible package or API changes,
```

Comment:
I would not agree that change is not needed. I strongly believe that SemVer needs this (or similar) clarification drastically.

People read SemVer spec
> 1. MAJOR version when you make incompatible API changes,

as
> I can change ANYTHING except the software contract (aka API) to avoid MAJOR version bump.

In my practice, inexperienced people where changing such things as: OS support, browser engine support, removed functionality, and most notably - runtime versions support. All that without major version bump.

It caused millions of $$ revenue loss for some companies.

You are saying - "could mean", but I'm saying - what it actually meant already.
Let's not put theories, instead let's look at past practice, at the events which actually happened.

I would agree that wording of this PR could be better. I welcome you to find better words.

Reply:
All those things are part of the software contract.

Reply:
Correct. Do you suggest to change the wording to...?

> 1. MAJOR version when you make incompatible software contract changes,

Reply:
I'm saying that that's what API means, and thus "API" is sufficient.

If people are going to misunderstand "API", then what's stopping them from misunderstanding "software contract"?

Reply:
"API" is a subset of "software contract". IMO.
"software contract" can also include such things as encoding, protocol(s), platform(s), package type, language(s), etc.

If you believe people would misunderstand "software contract" then please code up with better wording. Or, perhaps, a better PR.

Reply:
If you believe a further (third) discussion is needed - let's move to the #444